ui_testing/elements.py

```diff
@@ -543,7 +543,10 @@
         'test_date_filter': {'method': 'id',
                              'value': 'start_testDate'},
         'shipment_date_filter': {'method': 'id',
-                                 'value': 'start_shipmentDate'}
+                                 'value': 'start_shipmentDate'},
+        'analysis_tab': {
```

Comment:
when you removed this element from 'order' and wrote it in 'orders' Did you modified all functions that uses this element?

Reply:
No just a missing I wrote it first in the order page then I remembered that It should be in the orders page because those the filter related in the active table not belongs to the form to be in the order page and all test cases updated to be in orders